### PR TITLE
Add learner profiles and update authors

### LINKED ISSUE #23: Add learner profiles

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #23. [Here](https://nlesc.sharepoint.com/:w:/s/teamwork-sig/ER1P_QzOix1OrfvqqP4yvUMBD3F0-avoI2UsihHZEOMd1Q?e=vpYdJ2) is the old information. I only copied the learner profiles. 
For the rest this document holds information from before moving to Github. I think all information (i.e. learning objectives) is now in Github.

The learner profiles are still written very much according to the 'agile-only' course. We have to rewrite them in #24.

Also updates author information.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,3 +1,6 @@
-Sven van den Burg
+Sven van den Burg s.vanderburg@escience.nl
 Christiaan Meijer
-Berend Weel b.weel@esciencecenter.nl
\ No newline at end of file
+Berend Weel b.weel@esciencecenter.nl
+Fakhereh (Sarah) Alidoost
+Jaro Camphuijsen
+Robin Richardson
```
Comment (SarahAlidoost) [inline AUTHORS]:
```suggestion
Fakhereh (Sarah) Alidoost f.alidoost@esciencecenter.nl
Jaro Camphuijsen j.camphuijsen@esciencecenter.nl
Robin Richardson r.richardson@esciencecenter.nl
```

File: AUTHORS [hunk 2]
```diff
@@ -1,3 +1,6 @@
-Sven van den Burg
+Sven van den Burg s.vanderburg@escience.nl
 Christiaan Meijer
```
Comment (SarahAlidoost) [inline AUTHORS]:
```suggestion
Christiaan Meijer c.meijer@esciencecenter.nl
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
-# An Agile way of Working in eScience
-
-[![Create a Slack Account with us](https://img.shields.io/badge/Create_Slack_Account-The_Carpentries-071159.svg)](https://swc-slack-invite.herokuapp.com/)
-
-This repository generates the corresponding lesson website from [The Carpentries](https://carpentries.org/) repertoire of lessons.
+# Teamwork for Research Software
```
Comment (SarahAlidoost) [inline README.md]:
```suggestion
# Teamwork for Research Software Development
```

**Review by SarahAlidoost — APPROVED** [review]
@svenvanderburg thanks, I left minor suggestions on the changed files. For learner profiles, here are my comments:
- it would be nice if we address the issues of "how to create a team", "where to start", "what are team member characteristics", "what are the available tools (pros&cons)". For example, we might add something like this to Harold profile: "He would like to give advice to the hiring committee". or perhaps something like this to Agens: "She started working on one of the projects and is familiarizing herself with the way the center works. She wants to know how to create a team including other colleagues and project partners."
- I noticed that those three profiles are working at the center. This gives the impression that the lesson is designed for internal use at our center. right? if so, the profiles are fine. Otherwise, let's also add/change profiles who don't work at the center.

**Comment by svenvanderburg** [comment]
I agree with @SarahAlidoost's points. 
Indeed the course is intended for internal use as a start. But, in my opinion it would be useful to teach outside of the center as well (even if not taught by us, but by others). So I think we should make the learner profiles more generic.

@raar1 maybe you can rewrite the learner profiles a bit with these thoughts in mind. And in addition fix #24 straight away.

**Comment by raar1** [comment]
Sure, but I think I will merge this first, and then address all the suggestions on the profiles in a PR for #24

**Pull request merged**